src/qtt/measurements/new/signal_processor.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from qilib.data_set import DataSet
+
+from qtt.measurements.new.interfaces.signal_processor_interface import SignalProcessorInterface
+
+
+class SignalProcessor:
```

Comment:
I think the naming here is confusing. We have `SignalProcessorInterface`, and `SignalProcessor`. At first sight I expect a `SignalProcessor` to be an implementation of `SignalProcessorInterface`, but this is not the case. I would suggest renaming `SignalProcessor` to `SignalProcessorRunner` or something like that, so that the name actually describes what it is and avoids confusion.

Reply:
Agree. @CumulonimbusCalvus what do you think?